filebeat/prospector/redis/harvester.go

```diff
@@ -131,9 +131,7 @@ func (h *Harvester) Run() error {
 			"redis": common.MapStr{
 				"slowlog": subEvent,
 			},
-			"beat": common.MapStr{
-				"read_timestamp": common.Time(time.Now()),
-			},
+			"read_timestamp": common.Time(time.Now()),
```

Comment:
Yeah, that's how it is since we introduced the modules. We can still change it for 6.0..